index.html

```diff
@@ -8434,6 +8434,12 @@ <h2>Edge Computing</h2>
               node with lower latency and optionally under more direct control of the user (improving
               privacy). This can be important for control applications (e.g. in robotics), computer
               graphics (e.g. gaming) and for applications processing imagery (e.g. facial recognition).</li>

```

Comment:
```suggestion
              privacy). This can be important for control applications (e.g. in robotics), computer
              graphics (e.g., gaming) and for applications processing imagery (e.g., facial recognition).</li>

```